style.css

```diff
@@ -0,0 +1,170 @@
+#content {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  background-color: #191765;
+  font-family: 'Roboto', sans-serif;
+  background-image: url("images/Ellipse 18.png");
+  background-repeat: no-repeat;
+}
+
+#header-content {
+  height: 100px;
+  width: 100%;
+}
+
+.github img {
+  width: 40px;
+  height: 35px;
+}
+
+.arrow img {
+  margin-top: 1px;
+  height: 30px;
+  margin-right: 10px;
+}
+
+#menu {
+  display: flex;
+}
+
+#menu img {
+  margin-top: 1.5rem;
+  margin-right: 30px;
+}
+
+.name {
+  display: flex;
```

Comment:
- [ ] This is great using flexbox to layout your content, congrats 👏 , But we can still improve this code by using DRY (don't repeat yourself) but why? well, I realized that you have used the property `display: flex` at multiple places, please let's keep the code clean by extracting the property into a separate class that can be reused at multiple places, see the example below: 

- CSS file
```
.d-flex {
  display: flex;
}

.flex-center {
  align-items: center;
  justify-content: center;
}
```
- HTML file
```
<div class="d-flex flex-center">{your content}</div>
```

And remember you can do this for all the other properties that seem to be repeated multiple times, With this, I promise you'll be coding like a pro 😉